.gitlab-ci.yml

```diff
@@ -349,14 +344,24 @@ build:quick:
     when: always
 
 windows64:
-  extends: .windows-template
+  extends: .platform-template
   variables:
     ARCH: "64"
+  script:
+    - call dev/ci/platform-windows.bat
+  tags:
+    - windows-inria
+  except:
+    - /^pr-.*$/
```

Comment:
You don't want any platform test on pull requests?

Reply:
I thought that @SkySkimmer suggested this